tests/run.sh

```
@@ -27,6 +27,14 @@ else
 fi
 echo "::endgroup::"
 
+echo "::group::tests/php-floor.php (dereferenced `new` needs parentheses below PHP 8.4)"
```

Shellcheck (0.11.0)

[style] Use $(...) notation instead of legacy backticks `...`.

(SC2006)